src/main/java/com/thebois/models/beings/AbstractBeing.java

```diff
@@ -112,7 +130,44 @@ public void setRole(final AbstractRole role) {
 
     @Override
     public void update(final float deltaTime) {
-        move(deltaTime);
+
+        updateHunger(deltaTime);
+        updateHealth(deltaTime);
+        updateIsAlive();
```

Comment:
Shouldn't this be updated when health is updated?

Reply:
If all the `isAlive` does is check `health != 0` then you don't even need to update alive-state separately.